docs/management/watcher-ui/index.asciidoc

```diff
@@ -1,8 +1,56 @@
 [role="xpack"]
 [[watcher-ui]]
-== Watcher UI
+== Watcher
+
+Watcher is an {es} feature that you can use to create actions based on 
+conditions, which are periodically evaluated using queries on your data. 
+Watches are helpful for analyzing mission-critical and business-critical 
+streaming data. For example, you might watch application logs for performance 
+outages or audit access logs for security threats.
+
+With the Watcher UI, you can:
+
+* Create a simple threshold watch
+* View your watch history and action status
+* Acknowledge a watch to prevent too many executions
+* Edit, deactivate, and delete a watch
+* Create more advanced watches using API syntax
+
+[role="screenshot"]
+image:management/watcher-ui/images/watches.png["Watcher list"]
+
+{stack-ov}/xpack-alerting.html[Alerting on cluster and index events]
+is a good source for detailed 
+information on how watches work. If you are using the UI to create a 
+threshold watch, take a look at the different watcher actions. If you are 
+creating an advanced watch, you should be familiar with the parts of a 
+watch&#8212;input, schedule, condition, and actions.
+
+[float]
+[[watcher-security]]
+=== Required roles for Watcher
+
+To use Watcher, you must have the built-in `kibana_user` role.  
```

Comment:
It seems to me that this section only applies when security features are enabled, so it's a good idea to add that qualification. For example:

```suggestion
If the {es} {security-features} are enabled, to use {alert-features} in {kib}, you must have the built-in `kibana_user` role.  
```